### PR TITLE
Translates r2/keras/basic_classication.ipynb to PT-BR

### PR DESCRIPTION
**Pull request by brunapinos**
This pull request creates the first translation of TF2.0 Keras tutorial to Brazillian Portuguese. The first file is basic_classification.ipynb.

### CODE REVIEW

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,1007 @@
...
+        "    <td>8</td>\n",
+        "    <td>Bolsa (Bag)</td>\n",
+        "  </tr>\n",
+        "    <tr>\n",
+        "    <td>9</td>\n",
+        "    <td>Botas (Ankle boot)</td>\n",
+        "  </tr>\n",
+        "</table>\n",
+        "\n",
+        "Cada imagem é mapeada com um só label. Já que o *nome das classes* não são incluídas na base de dados, os armazenes aqui para usá-los mais tarde quando plotarmos as imagens:"
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: "os armazenes aqui" não soa natural em pt_br. Poderia melhorar esse trecho?

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,1007 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "TMPI88iZpO2T"
+      },
+      "source": [
+        "Existem 10000 images in the test set. Again, each image is represented as 28 x 28 pixels:"
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: Faltou traduzir este trecho

File: site/pt/r2/tutorials/keras/basic_classification.ipynb (hunk 1 above)
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: "os armazenes aqui" não soa natural em pt_br. Poderia melhorar esse trecho?

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,1007 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Ee638AlnCaWz"
+      },
+      "source": [
+        "Para verificar que os dados estão no formato correto e que estamos prontos para construir e treinar a rede, vamos mostrar as primeiras 25 imagens do *conjunto de treinamento* e mostrar o nome das classes de cada imagens abaixo."
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: **"classes de cada imagens abaixo"**, substiuir por **"classes de cada imagem abaixo"**. A palavra **imagem** deve estar no singular nesse trecho.

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,1007 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Gxg1XGm0eOBy"
+      },
+      "source": [
+        "### Montar as camadas\n",
+        "\n",
+        "O principal bloco de construção da rede neural é a camada (*layer*). As camadas (*layers*) extrai representações dos dados inseridos na rede. Com sorte, essas representações são significativos para o problema à mão.\n",
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: - "As camadas **extraem**"
- "essas representações são **significativas** para o problema."

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 5]
```diff
@@ -0,0 +1,1007 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "gut8A_7rCaW6"
+      },
+      "source": [
+        "A primeira camada da rede, `tf.keras.layers.Flatten`, transforma o formato da imagem de um array de imagens de duas dimensões (of 28 by 28 pixels) para um array de uma dimensão (de 28 * 28 = 784 pixels). Pense nessa camada como camadas não empilhadas de pixels de uma imagem e os emfilere. Essa camada não tem paramêtros para aprender; ela só reformata os dados.\n",
+        "\n",
+        "Depois dos pixels serem achatados, a rede consite de uma sequência de duas camadas `tf.keras.layers.Dense`. Essa são camadas neurais *densely connected*, ou *fully connected*. A primeira camada `Dense` tem 128 nós (or neurônios). A segunda (e última) camda é uma *softmax*  de 10 nós que retorna um array de 10 probabilidades, cuja soma resulta em 1. Cada nó contem um valor que indica a probabilidade de que aquela imagem pertence a uma das 10 classes.\n",
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: Mudar para "(**ou** neurônios)"

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 6]
```diff
@@ -0,0 +1,1007 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "EQ5wLTkcCaXY"
+      },
+      "source": [
+        "Agora prediresmo a label correta para essa imagem:"
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: O correto seria "**prediremos**"

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 7]
```diff
@@ -0,0 +1,1007 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "cU1Y2OAMCaXb"
+      },
+      "source": [
+        "`model.predict` returna a lista de listas—uma lista para cada imagem em um *batch* de dados. Pegue a predição de nossa (única) imagem no *batch*:"
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: Corrigir para "**retorna** a lista..."

File: site/pt/r2/tutorials/keras/basic_classification.ipynb [hunk 8]
```diff
@@ -0,0 +1,1007 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "YFc2HbEVCaXd"
+      },
+      "source": [
+        "E, como antes, o modelo preveu a label como 9."
```
Comment (diogoarm) [inline site/pt/r2/tutorials/keras/basic_classification.ipynb]: Corrigir para "E, como antes, o modelo **previu** a label..."
